include/versioningCompiler/CompilerImpl/ClangLLVM/NewPMDriver.h

```diff
@@ -16,6 +16,7 @@
 /// opt.cpp.
 ///
 //===----------------------------------------------------------------------===//
+// Added macro comments to make it work from LLVM 13 to LLVM 16
 
 #ifndef LIB_VERSIONING_COMPILER_CLANG_LLVM_TOOLS_OPT_NEWPMDRIVER_H
 #define LIB_VERSIONING_COMPILER_CLANG_LLVM_TOOLS_OPT_NEWPMDRIVER_H
```

Comment:
I kept it identical to the original file name, so that it can be easily referenced.
https://github.com/llvm/llvm-project/blob/main/llvm/tools/opt/NewPMDriver.h

Reply:
Ok, makes sense.
However, please add a comment right near the top to specify this